packages/fuels-core/src/code_gen/abigen.rs

```diff
@@ -668,42 +668,42 @@ mod tests {
     #[test]
     fn test_abigen_struct_inside_enum() {
         let contract = r#"
-[
-  {
-    "type": "function",
-    "inputs": [
-      {
-        "name": "b",
-        "type": "enum Bar",
-        "components": [
+        [
```

Comment:
Yes sorry ,I should have mention it. I did not change the test I just indented the string to match the other test. Also, I heavily use folding inside my editor and because those string were not indented It would mess up the folding. I would like to leave it like this.